libraries/ms-helper/inc/thermistor.h

```diff
@@ -33,3 +33,12 @@ StatusCode thermistor_get_temp(ThermistorStorage *storage, uint16_t *temperature
 
 // Calculate the temperature in deciCelsius from ohms
 StatusCode thermistor_calculate_temp(uint32_t thermistor_resistance_ohms, uint16_t *temperature_dc);
+
+// Finds the resistance of the thermistor at a certain temperature
+StatusCode thermistor_calculate_resistance(uint16_t temperature_dc,
+                                           uint16_t *thermistor_resistor_ohms);
+
+// Finds the node voltage given a temperature
+// Requires the supply voltage in units of 100 microvolts and return the node voltage in millivolts
```

Comment:
This really shouldn't be here. Since it's just used in plutus, just make it a simple function in fault_monitor.